app/models/concerns/avatarable.rb

```diff
@@ -6,5 +6,15 @@ module Avatarable
 
     validates :avatar, content_type: ["image/png", "image/jpeg", "image/jpg"],
       max_file_size: 2.megabytes
+
+    validate :avatar_validation, on: :create
```

Comment:
Interesting approach! Is there not a built in validator to do this?

If not, we should create a custom validator. Check out two lines above which uses `ContentTypeValidator`.

Reply:
Looked this up and this is what I found.However, I can do more research to see if there's a built-in validator and how to make custom validations. Its been a fun experience!

Reply:
Hey, added two new commits. On the console they seem to be passing the validation but fails the test when i try to push. Kindly advise